pkg/controller/certificaterequests/acme/acme_test.go

```diff
@@ -72,6 +73,34 @@ func generateCSR(t *testing.T, secretKey crypto.Signer, commonName string, dnsNa
 	return csr
 }
 
+func generateCSRWithIPs(t *testing.T, secretKey crypto.Signer, commonName string, dnsNames []string, ips []string) []byte {
+	// The CommonName of the certificate request must also be present in the DNS
+	// Names.
+
+	certIPs := []net.IP(nil)
```

Comment:
Nit:

```suggestion
	var certIPs []net.IP
```

...is how I'd initialise a nil slice (zero value), but I was just curious why do it this way? Is it a goland thing?

Reply:
I like it this way but happy to change